chatbot-conversational_AI/knowledge-database/api-mssql-function/k8s/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  name: api-mssql-function
+spec:
+  gateway: kyma-gateway.kyma-system.svc.cluster.local
+  rules:
+    - path: /.*
+      accessStrategies:
+        - config: {}
+          handler: noop
+      methods:
+        - GET
+        - POST
+        - DELETE
+  service:
+    host: api-mssql-function
+    name: api-mssql-function
+    port: 80
```

Comment:
add newlines. check in other places also

Reply:
I don't get where I should add new lines in the yaml files, sorry. I basically copied the style from Jamie. See e.g. https://github.com/SAP-samples/kyma-runtime-extension-samples/blob/main/api-mssql-function/k8s/deployment.yaml